ui/apps/platform/src/Containers/Clusters/DelegateScanning/Components/DelegatedScanningSettings.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Card, CardBody, Flex, FlexItem, Radio } from '@patternfly/react-core';
+
+import FormLabelGroup from 'Components/PatternFly/FormLabelGroup';
+import { EnabledSelections } from 'types/dedicatedRegistryConfig.proto';
+
+type DelegatedScanningSettingsProps = {
+    enabledFor: EnabledSelections;
+    onChangeEnabledFor: (EnabledSelections) => void;
```

Comment:
Just missing the parameter name here. Without it I don't think typescript would complain if you did something like `onChangeEnabledFor('blah')`
```suggestion
    onChangeEnabledFor: (newEnabledState: EnabledSelections) => void;
```